symengine/tests/cwrapper/test_cwrapper.c

```diff
@@ -63,6 +63,15 @@ void test_cwrapper()
     SYMENGINE_C_ASSERT(strcmp(s, "y*(456 + x)/z") == 0);
     basic_str_free(s);
 
+    basic numer, denom;
+    numer_denom_as_numer_denom(e, numer, denom);
+    s = basic_str(numer);
+    SYMENGINE_C_ASSERT(strcmp(s, "y*(456 + x)") == 0);
```

Comment:
It's better if you can just check that `e` is equal to `y*(456 + x)` using `basic_eq` instead of comparing strings.